index.js

```diff
@@ -13,21 +15,32 @@ if(process.env.NODE_ENV !== 'local') {
 }
 
 app.use(s3o);
+app.use(express.static(path.join(__dirname + '/project')));
 
 app.get('/keysFor/:project', (req, res) => {
 	const validUser = checkUser(req.cookies.s3o_username);
+	let response;
+
 	if(validUser) {
 		const keys = formatKeys(req.params.project);
 		const hasKeys = keysExist(keys);
 
 		if(hasKeys) {
-			return res.json({'key': process.env[keys.key], 'secret': process.env[keys.secret]});
+			response = {'key': process.env[keys.key], 'secret': process.env[keys.secret]};
 		} else {
-			return res.json({'error': 'The keys for this resource don\'t exist.'});
+			response = {'error': 'The keys for this resource don\'t exist.', 'errorType': '404'};
 		}
+	} else {
+		response = {'error': 'You are not allowed to get keys for this resource.', 'errorType': '403'};
+	}
+
+	if(response.key !== undefined) {
+		return copy(JSON.stringify(response), () => {
+			return res.sendFile(path.join(__dirname + '/project/' + req.params.project + '.html'));
+		});
 	}
 
-	return res.json({'error': 'You are not allowed to get keys for this resource.'});
+	res.status(response.errorType).send(response.error);
```

Comment:
You may need/want to set the `Content-Type` headers if `res.json()` is no longer being used.

Reply:
Sending a string back, not JSON.